scipy/spatial/transform/rotation.py

```diff
@@ -1463,6 +1465,88 @@ def magnitude(self):
         else:
             return angles
 
+    def reduce(self, left=None, right=None, return_indices=False):
+        """Reduce this rotation with the provided rotation groups.
+
+        Parameters
+        ----------
+        left : `Rotation` instance, optional
+            Object containing the left rotation(s). Default value (`None`)
+            corresponds to the identity rotation.
+        right : `Rotation` instance, optional
+            Object containing the right rotation(s). Default value (`None`)
+            corresponds to the identity rotation.
+        return_indices : bool
+            Whether to return the best indices
+
+        Returns
+        -------
+        reduced : `Rotation` instance
+            Object containing reduced rotations.
+        left_best, right_best: integer ndarray
+            Indices of elements from `left` and `right` used for reduction.
+        """
+        if left is None and right is None:
+            return self.__class__(self._quat, normalized=True, copy=True)
+        elif right is None:
+            right = Rotation.identity()
+        elif left is None:
+            left = Rotation.identity()
+
+        # Levi-Civita tensor for triple product computations
+        e = np.zeros((3, 3, 3))
+        e[0, 1, 2] = e[1, 2, 0] = e[2, 0, 1] = 1
+        e[0, 2, 1] = e[2, 1, 0] = e[1, 0, 2] = -1
+
+        """
```

Comment:
Perhaps use a standard comment with # and plain formatting. 

It's just uncommon to use literal docstring for in-code commentary.

Reply:
Fixed.